x-pack/plugins/observability_solution/observability_ai_assistant_app/server/functions/visualize_esql.ts

```diff
@@ -38,16 +60,16 @@ export function registerVisualizeESQLFunction({
           name,
           meta: { type: esFieldTypeToKibanaFieldType(type) },
         })) ?? [];
+
+      const message = getSystemMessage(intention, query, errorMessages);
```

Comment:
nit, but a system message is the first message in a conversation, that modifies the behaviour of the LLM for the entire conversation. it's a bit of a technical concept. probably something like `getMessageForLLM` is more appropriate here.

Reply:
done 05398aceb2965f205104dd2dc30783b8c258702a